compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -4523,6 +4555,50 @@ private STNode parseObjectTypeQualifiers() {
         return parseObjectTypeNextQualifiers(firstQualifier);
     }
 
+    /**
+     /**
+     * Parse object constructor expression type qualifiers.
+     *
+     * @return Parsed qualifier list with only client or empty node list
+     */
+    private STNode parseObjectConstructorExpressionQualifier() {
+        STNode qualifier;
+        STToken nextToken = peek();
+        switch (nextToken.kind) {
+            case CLIENT_KEYWORD:
+                qualifier = parseClientKeyword();
```

Comment:
Since we check if the next token is a `CLIENT_KEYWORD`(in this switch case) and reach here don't need to check it again inside `parseClientKeyword()` right?. `consume()` would do.